tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+bok_choy==0.3.2
```

Comment:
@mtyaka That makes sense. You might want to also take a look at my changes on our fork of this, where I've set up Travis CI on my ongoing PR: https://github.com/open-craft/xblock-mentoring/pull/1 . It has a consistent test environment but it does so just by using whatever the pinned versions in xblock-sdk are.